gemac/management.py

```diff
@@ -37,12 +36,12 @@ def management(hostintf, mdiointf, reset):
     Args:
         hostintf - Instance of 'HostManagementInterface' class in interfaces.
         mdiointf - Instance of 'MDIOInterface' class in interfaces.
+        configregs - List of 10 32-bits wide Configuration Registers.
+        addrtable - List of 4 48-bits wide MAC Addresses to be used by
+            Address Filter.
         reset: Asynchronous reset Signal from Host
```

Comment:
So you suggesting me to replace
    reset: Asynchronous reset Signal from Host
with
    reset: System reset
right?